app/__init__.py

```diff
@@ -44,19 +44,61 @@ def products():
     if ("seller_email" not in session):
         return redirect('/login')
     products = Product.query.all()
-    print(products)
     return render_template('seller/products.html', products=products)
 
 @app.route('/products', methods=['POST'])
 def add_product():
     if ("seller_email" not in session):
         return redirect('/login')
-    print(request.form)
     new_product = Product(
         name=request.form['product_name'],
         price=float(request.form['product_price']),
         image_url=request.form['product_image'],
         category=request.form["product_selling_status"])
     db.session.add(new_product)
     db.session.commit()
-    return redirect('/products') 
\ No newline at end of file
+    return redirect('/products') 
+
+@app.route('/products/<id>', methods=['GET'])
+def get_product(id):
+    if ("seller_email" not in session):
+        return redirect('/login')
+    product = Product.query.get_or_404(id)
+    return render_template('seller/product.html', prod = product)
```

Comment:
@ethanschreur How is this related to editing products ?

Reply:
That is the route that leads to where the products can be edited.

Reply:
> @ethanschreur How is this related to editing products ?

That is the route that leads to where the products can be edited.

Reply:
That's ok